test/extensions/tracers/opentelemetry/config_test.cc

```diff
@@ -41,6 +41,27 @@ TEST(OpenTelemetryTracerConfigTest, OpenTelemetryHttpTracer) {
   EXPECT_NE(nullptr, opentelemetry_tracer);
 }
 
+TEST(OpenTelemetryTracerConfigTest, OpenTelemetryHttpTracerNoExporter) {
+  NiceMock<Server::Configuration::MockTracerFactoryContext> context;
+  context.server_factory_context_.cluster_manager_.initializeClusters({"fake_cluster"}, {});
+  OpenTelemetryTracerFactory factory;
+
+  const std::string yaml_string = R"EOF(
+    http:
+      name: envoy.tracers.opentelemetry
+      typed_config:
+        "@type": type.googleapis.com/envoy.config.trace.v3.OpenTelemetryConfig
+  )EOF";
+  envoy::config::trace::v3::Tracing configuration;
+  TestUtility::loadFromYaml(yaml_string, configuration);
+
+  auto message = Config::Utility::translateToFactoryConfig(
+      configuration.http(), ProtobufMessage::getStrictValidationVisitor(), factory);
+  auto opentelemetry_tracer = factory.createTracerDriver(*message, context);
+  std::cout << typeid(opentelemetry_tracer).name() << std::endl;
```

Comment:
Remove?

Reply:
https://github.com/envoyproxy/envoy/pull/24792/commits/6657bb606bc042c08494ed47fe7f2cb9472ed40a

thanks for the review!